cmd/process.go

```diff
@@ -74,15 +74,20 @@ func process(ctx context.Context, logger *zap.SugaredLogger) error {
 		logger.Fatal("failed to initialize new server", zap.Error(err))
 	}
 
+	conn, err := events.NewConnection(config.AppConfig.Events, events.WithLogger(logger))
+	if err != nil {
+		logger.Fatalw("failed to initialize events", "error", err)
+	}
+
 	server := &server.Server{
-		Context:          cx,
-		Debug:            viper.GetBool("logging.debug"),
-		Echo:             eSrv,
-		Locations:        viper.GetStringSlice("event-locations"),
-		Logger:           logger,
-		SubscriberConfig: config.AppConfig.Events.Subscriber,
-		ChangeTopics:     viper.GetStringSlice("change-topics"),
-		IPBlock:          viper.GetString("ipblock"),
+		Context:      cx,
+		Debug:        viper.GetBool("logging.debug"),
+		Echo:         eSrv,
+		Locations:    viper.GetStringSlice("event-locations"),
+		Logger:       logger,
+		Connection:   conn,
```

Comment:
Give the `Connection` a more detailed name? Maybe `EventsConnection`

Reply:
✅